packages/shared/src/types/props/banner.ts

```diff
@@ -49,6 +49,7 @@ export interface BannerProps {
     email?: string;
     fetchEmail?: () => Promise<string | null>;
     separateArticleCount?: boolean;
+    digisubPrice?: string;
```

Comment:
as discussed IRL - let's make this an object in case we add other products in future:
```
interface ProductPrices {
  digisub: string;
}
```

Reply:
Done